apps/io-wallet-user-func/src/infra/azure/storage/queue.ts

```diff
@@ -26,7 +26,7 @@ const sendMessage =
   > =>
   ({ queueClient }) =>
     pipe(
-      TE.tryCatch(() => queueClient.sendMessage(message), E.toError),
+      TE.tryCatch(async () => queueClient.sendMessage(message), E.toError),
       TE.filterOrElse(
```

Comment:
why did you add `async`?

Reply:
If I don't use the async keyword, the unit test labeled "should return a HTTP 204 response on success" fails. This is probably just a vitest issue, but it seems necessary to pass all unit tests.

Reply:
I think the unit tests are wrong